tests/test_rooms/test_monthly_meetings_duration.py

```diff
@@ -0,0 +1,20 @@
+import json
+from tests.base import BaseTestCase
+from fixtures.token.token_fixture import admin_api_token
+from fixtures.room.room_monthly_meeting_duration_fixtures import (
+    get_monthly_meetings_total_duration_query,
+    get_monthly_meetings_total_duration_response
+)
+
+
+class TotalMonthlyDurations(BaseTestCase):
+
+    def test_total_monthly_meeting_durations(self):
```

Comment:
You would now need to alter your tests and fixtures to conform to the new  format of tests and fixtures in order to remove duplicates

Reply:
Ok. Let me put that into consideration.